lib/resources/pip.rb

```diff
@@ -62,18 +63,52 @@ def to_s
 
     private
 
+    def cmd
+      @__cmd ||= inspec.command("#{@pip_cmd} show #{@package_name}")
+    end
+
+    def cmd_successful?
+      result = inspec.command("#{@pip_cmd} show #{@package_name}")
+      return true if result.exit_status == 0
+
+      if result.exit_status != 0
+        # If pip on windows is not the latest, it will create a stderr value along with stdout
+        # Example:
+        #   stdout: "Name: Jinja2\r\nVersion: 2.10..."
+        #   stderr: "You are using pip version 9.0.1, however version 9.0.3 is available..."
+        if inspec.os.windows? && !result.stdout.empty?
+          return true
+        end
+      end
+
+      false
+    end
+
+    # Paths of Python and Pip
+    # {"Pip" => nil, "Python" => "/path/to/python"}
+    #
+    # @return [Hash] of paths
+    def paths
```

Comment:
You mind updating this to `windows_paths`. Since its windows only command it may read better.

Reply:
@jquick good catch! Thanks!